internal/perf/cpu.go

```diff
@@ -44,7 +43,11 @@ func HandleCPUProfileSignals() {
 		}()
 
 		// Profile.
-		pprof.StartCPUProfile(f)
+		err = pprof.StartCPUProfile(f)
+		if err != nil {
+			errorLogger := logger.NewError("")
+			errorLogger.Printf("StartCPUProfile failed: %v", err)
```

Comment:
you should create a method like this to log errors incase error is not available on available logger. https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/internal/logger/logger.go#L138

Creating a newError logger for every log is not correct way to do.

Reply:
Updated! Thanks!